src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -216,6 +219,11 @@ function MoneyRequestConfirmationList({
     const [policyCategoriesDraft] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY_CATEGORIES_DRAFT}${policyID}`);
     const [lastSelectedDistanceRates] = useOnyx(ONYXKEYS.NVP_LAST_SELECTED_DISTANCE_RATES);
     const [currencyList] = useOnyx(ONYXKEYS.CURRENCY_LIST);
+    const [betas] = useOnyx(ONYXKEYS.BETAS);
+    const {shouldShowProductTrainingTooltip, renderProductTrainingTooltip} = useProductTrainingContext(
+        CONST.PRODUCT_TRAINING_TOOLTIP_NAMES.SCAN_TEST_CONFIRMATION,
+        Permissions.canUseManagerMcTest(betas) && selectedParticipantsProp.some((participant) => participant.login === CONST.EMAIL.MANAGER_MCTEST),
```

Comment:
Have you considered moving this part `selectedParticipantsProp.some((participant) => participant.login === CONST.EMAIL.MANAGER_MCTEST)` into a separate util? I see it's used more than once, so maybe that would make sense, wdyt?